CMakeLists.txt

```diff
@@ -479,6 +512,12 @@ add_subdirectory(core)
 # Note that rocprofiler-register is in base and is what higher level clients
 # depend on. The profiler itself is independent.
 add_subdirectory(profiler)
+# Data center tools (RDC, etc.)
+if(THEROCK_ENABLE_DC_TOOLS AND THEROCK_ENABLE_RDC AND NOT WIN32)
```

Comment:
Prefer to gate the inclusion of the subdirectory on THEROCK_ENABLE_DC_TOOLS and then condition specific subprojects on THEROCK_ENABLE_RDC and WIN32. That way is it obvious locally what is built where without tracking up to the top level.